django/core/management/commands/loaddata.py

```diff
@@ -253,8 +253,14 @@ def fixture_dirs(self):
                     "and cannot be listed in settings.FIXTURE_DIRS." % (app_dir, app_label)
                 )
 
-            if self.app_label and app_label != self.app_label:
-                continue
+            if self.app_label:
+                # Passing app_label as a set is a temporary hack for Django 1.8
+                # to speedup Django's test suite on MSSQL.
+                if isinstance(self.app_label, set):
```

Comment:
Maybe adding `and os.environ.get('BULK_LOAD_INITIAL_DATA')` data is a practical solution?